WGS_pipeline/guide_mutation_binary.wdl

```diff
@@ -95,3 +111,66 @@ task guide_mutation {
         author: "Simone Zhang"
     }
 }
+
+task guide_mutation_intersect {
+    input {
+        String sample_id
+        String docker="biocontainers/bedtools:v2.28.0_cv2"
+
+        File avana_in_vcf_bed
+        File humagne_in_vcf_bed
+        File ky_in_vcf_bed
+        # File brunello_in_vcf_bed
+        # File TKOv3_in_vcf_bed
+
+        File avana_bed = "gs://ccleparams/avana_guides.bed"
+        File humagne_bed = "gs://ccleparams/humagne_guides.bed"
+        File ky_bed = "gs://ccleparams/ky_score_guides.bed"
+        # File brunello_bed = "gs://ccleparams/brunello_guides.bed"
+        # File TKOv3_bed = "gs://ccleparams/tkov3_guides.bed"
+    
+        Int memory = 4
+        Int boot_disk_size = 10
+        Int num_threads = 1
+        Int num_preempt = 5
+        Int disk_space = 40
+    }
+
+    command <<<
+        set -euo pipefail
+
+        bedtools intersect -a ~{avana_bed} -b ~{avana_in_vcf_bed} -c > ~{sample_id}_avana_mut_binary.bed
+        bedtools intersect -a ~{humagne_bed} -b ~{humagne_in_vcf_bed} -c > ~{sample_id}_humagne_mut_binary.bed
+        bedtools intersect -a ~{ky_bed} -b ~{ky_in_vcf_bed} -c > ~{sample_id}_ky_mut_binary.bed
+        # bedtools intersect -a ~{brunello_bed} -b ~{brunello_in_vcf_bed} -c > ~{sample_id}_brunello_mut_binary.bed
+        # bedtools intersect -a ~{TKOv3_bed} -b ~{TKOv3_in_vcf_bed} -c > ~{sample_id}_tkov3_mut_binary.bed
+
+        bedtools sort -i ~{sample_id}_avana_mut_binary.bed > ~{sample_id}_avana_mut_binary.sorted.bed
+        bedtools sort -i ~{sample_id}_humagne_mut_binary.bed > ~{sample_id}_humagne_mut_binary.sorted.bed
+        bedtools sort -i ~{sample_id}_ky_mut_binary.bed > ~{sample_id}_ky_mut_binary.sorted.bed
+        # bedtools sort -i ~{sample_id}_brunello_mut_binary.bed > ~{sample_id}_brunello_mut_binary.sorted.bed
+        # bedtools sort -i ~{sample_id}_tkov3_mut_binary.bed > ~{sample_id}_tkov3_mut_binary.sorted.bed
+        
+    >>>
+
+    output {
+        File avana_binary_mut="~{sample_id}_avana_mut_binary.sorted.bed"
+        File humagne_binary_mut="~{sample_id}_humagne_mut_binary.sorted.bed"
+        File ky_binary_mut="~{sample_id}_ky_mut_binary.sorted.bed"
+        # File brunello_binary_mut="~{sample_id}_brunello_mut_binary.sorted.bed"
+        # File tkov3_binary_mut="~{sample_id}_tkov3_mut_binary.sorted.bed"
+    }
+
+    runtime {
+        docker: docker
+        bootDiskSizeGb: "~{boot_disk_size}"
```

Comment:
No need to customize bootDiskSizeGb, especially since the minimum/default is actually higher than 10 GB now.